askama_derive/src/generator.rs

```diff
@@ -734,7 +734,7 @@ impl<'a> Generator<'a> {
             }
         };
 
-        mem::drop(mem::replace(&mut self.buf_writable, current_buf));
+        mem::drop(mem::replace(&mut self.buf_writable.buf, current_buf));
```

Comment:
This line could simply be `self.buf_writable.buf = current_buf;` and be much easier to read. Maybe you could sneak that change into the PR? :)

Reply:
Definitely!